server/src/main/java/org/opensearch/index/shard/IndexShard.java

```diff
@@ -4656,10 +4656,9 @@ public void syncTranslogFilesFromRemoteTranslog() throws IOException {
     /**
      * Downloads segments from remote segment store.
      * @param overrideLocal flag to override local segment files with those in remote store
-     * @param refreshLevelSegmentSync last refresh checkpoint is used if true, commit checkpoint otherwise
      * @throws IOException if exception occurs while reading segments from remote store
      */
-    public void syncSegmentsFromRemoteSegmentStore(boolean overrideLocal, boolean refreshLevelSegmentSync) throws IOException {
+    public void syncSegmentsFromRemoteSegmentStore(boolean overrideLocal) throws IOException {
```

Comment:
Is the removal of `refreshLevelSegmentSync` param intentional?

Reply:
No, all the callers are used `true` anyway.